packages/editor/src/Combobox.tsx

```diff
@@ -27,7 +27,7 @@ const keys = {
   13: 'return',
 }
 
-export default ({
+const Combobox = ({
```

Comment:
Note: I'm changing all the `default export () => {}` to `const Foo = () => {}; default export Foo` due to gatsbyjs/gatsby#23575.